generator/locality_sorter.cpp

```diff
@@ -24,21 +24,58 @@
 
 #include <cstdint>
 #include <fstream>
+#include <functional>
 #include <limits>
 #include <set>
 #include <vector>
 
 using namespace feature;
 using namespace std;
 
-class LocalityCollector : public FeaturesCollector
+namespace
 {
-  DISALLOW_COPY_AND_MOVE(LocalityCollector);
+FeatureBuilder2 & GetFeatureBuilder2(FeatureBuilder1 & fb)
```

Comment:
s/GetFeatureBuilder2/ToFeatureBuilder2/

But do you really need this as a function?

Reply:
done